frontend/src/patientApp/timeOfTest/DOB.tsx

```diff
@@ -154,31 +150,24 @@ const DOB = () => {
       <div className="grid-container maxw-tablet">
         <p className="margin-top-3">{t("testResult.dob.enterDOB2")}</p>
         <p>{t("testResult.dob.linkExpirationNotice")}</p>
-        <TextInput
+        <DateInput
           className="width-mobile"
           label={t("testResult.dob.dateOfBirth")}
           name={"birthDate"}
-          type={birthDateHidden ? "password" : "text"}
-          autoComplete={"on"}
-          value={birthDate}
-          ariaDescribedBy={"bdayFormat"}
-          hintText={t("testResult.dob.format")}
-          onBlur={validateBirthDate}
+          monthName={"birthMonth"}
+          dayName={"birthDay"}
+          yearName={"birthYear"}
+          monthValue={birthMonth}
+          dayValue={birthDay}
+          yearValue={birthYear}
+          monthOnChange={(evt: any) => setBirthMonth(evt.currentTarget.value)}
+          dayOnChange={(evt: any) => setBirthDay(evt.currentTarget.value)}
+          yearOnChange={(evt: any) => setBirthYear(evt.currentTarget.value)}
```

Comment:
:ok_hand:

Reply:
can we specify the type to number

Reply:
Do you meant the type of the input or the event type?